src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Random randomColor = new Random();
```

Comment:
We can add field with type `Random` to our class, so we won't need to create an instance every time when method invoked
also, let's find a better name for `randomColor` variable, because it isn't a color) 
(just `random` will be ok)

Reply:
Got it